setup.py

```diff
@@ -503,7 +503,7 @@ def do_setup():
         # DEPENDENCIES_EPOCH_NUMBER in the Dockerfile
         #####################################################################################################
         setup_requires=[
-            'docutils>=0.14, <1.0',
+            'docutils>=0.14, <0.16'
```

Comment:
What changed in 0.16?

Reply:
One of our dependencies is incompatible with 0.16 and when you try it on a fresh system you get "incompatibility" problem (not sure which one) - until we fix it with pinnig - it's the simplest way to make it works out-of the box.